content/applications/finance/accounting/reporting/intrastat.rst

```diff
@@ -2,27 +2,173 @@
+:guilabel:`Save`.
+
+.. _intrastat/product-configuration:
+
+Product configuration
+=====================
+
+All products must be properly configured to be included in the Intrastat report.
+
+.. _intrastat/commodity-code:
+
+Commodity code
+--------------
+
+Commodity codes are internationally recognized reference numbers used to classify goods depending on
+their **nature**. Intrastat uses the `Combined Nomenclature <https://taxation-customs.ec.europa.eu/customs-4/calculation-customs-duties/customs-tariff/combined-nomenclature_en>`_.
+
+To add a commodity code, go to :menuselection:`Accounting --> Customers --> Products` and select a
+product. Under the :guilabel:`Accounting` tab, set the product's :guilabel:`Commodity Code`.
+
+.. seealso::
+   `National Bank of Belgium - Intrastat commodity codes
+   <https://www.nbb.be/en/statistics/foreign-trade/nomenclature-and-codes>`_
+
+.. _intrastat/origin-country:
+
+Country of origin
+-----------------
+
+To add the product's country of origin, go to :menuselection:`Accounting --> Customers --> Products`
+and select a product. Under the :guilabel:`Accounting` tab, set the :guilabel:`Country of Origin`.
+
+.. _intrastat/invoice-bill-configuration:
+
+Invoices and bills configuration
+================================
+
+Once products are properly configured, several settings must be configured on the invoices and bills
+you create.
+
+.. _intrastat/transaction-code:
+
+Transaction code
+----------------
+
+Transaction codes are used to identify a transaction's nature. To set a transaction code on an
+invoice line, create an invoice or bill, click the :guilabel:`⋮` button, tick :guilabel:`Intrastat`,
+and use the newly-added :guilabel:`Intrastat` column to select a transaction code.
+
+.. image:: intrastat/intrastat-ellipsis.png
+   :align: center
+   :alt: Adding the Intrastat column to an invoice or bill
+
+.. seealso::
+   `National Bank of Belgium - Intrastat: Nature of transactions from January 2022
+   <https://www.nbb.be/doc/dd/onegate/data/new_natures_of_transaction_2022_en.pdf>`_
+
+.. _intrastat/partner-country:
+
+Partner country
+---------------
+
+The partner country represents the vendor's country in case of bills, and the customer's country in
+case of invoices. It is automatically filled in using the :guilabel:`Country` field in the **contact
+form card** of the partner.
+
+To edit the partner country manually, when on an invoice or bill, click the :guilabel:`Other Info`
+tab, and select the :guilabel:`Intrastat Country`.
+
+.. _intrastat/transport-code:
+
+Transport code
+~~~~~~~~~~~~~~
+
+The transport code identifies the presumed **mode of transport** used to send the goods (arrival or
+dispatch). Once you have selected a :ref:`partner country <intrastat/partner-country>`, the
+:guilabel:`Intrastat Transport Mode` becomes available underneath it.
+
+.. _intrastat/value:
+
+Value of the goods
+------------------
```

Comment:
change h level (?)

Reply:
I still think it makes more sense as such